_layouts/blog.html

```diff
@@ -36,7 +36,25 @@ <h1><a href="/blog/">Trino blog</a></h1>
     </div>
   </div>
 
-  {{ content }}
+  <!-- Avoids showing table of contents on blog index page -->
+  {% if content contains '<!-- Blog Index -->' %}
```

Comment:
Lets make this independent of the page we use it on by using a value like "no-right-nav"  or so

Reply:
IMO that's an easy change to make if/when we decide we want to remove the nav from a post at some point. But if it's only one page for now, we can call that page out specifically for better clarity.

Reply:
ok